features/taxation/managing_tax_rates/tax_rates_filtration/filtering_tax_rates_by_end_date.feature

```diff
@@ -0,0 +1,36 @@
+@managing_tax_rates
+Feature: Filtering tax rates by end date
+    In order to see tax rates from an end date range
+    As an Administrator
+    I want to be able to filter tax rates on the list
+
+   Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "2022 tax rate" tax rate of 50% for "Clothes" within the "US" zone with dates between "2022-01-01" and "2022-12-31"
+        And the store has "2023 tax rate" tax rate of 15% for "Clothes" within the "US" zone with dates between "2023-01-01" and "2023-12-31"
+        And the store has "3 weeks tax rate" tax rate of 25% for "Clothes" within the "US" zone with dates between "2022-12-24" and "2023-01-15"
+        And I am logged in as an administrator
+
+    @ui @api
```

Comment:
These scenarios should not be marked as `@api` as they're not implemented in API 🖖

Reply:
So are they implemented or not? Because they are merged now. :)

Reply:
Never mind, I'm about to implement admin tax rate apis and those scenarios https://github.com/Sylius/Sylius/pull/14593